lms/templates/courseware/course_about_sidebar_header.html

```diff
@@ -65,4 +66,6 @@
       </a>
   </div>
   % endif
+</header>    </a>
+  </div>
```

Comment:
These extra closing tags should probably be removed.

Reply:
Hmm, I don't know who sneaked those in there! 

I removed them and amended my original commit to keep the Git history clean.